diff --git a/cmake-tool/init-build.sh b/cmake-tool/init-build.sh
index c7a6638..c8cbf1a 100755
--- a/cmake-tool/init-build.sh
+++ b/cmake-tool/init-build.sh
@@ -31,19 +31,21 @@ then
     exit 1
 fi
 
-if [ -d "$HOME/.sel4_cache" ]
-then
-    CACHE_DIR="$HOME/.sel4_cache"
-else
-    CACHE_DIR="$SCRIPT_PATH/.sel4_cache"
-fi
+# MSKORDAL: commented cache
+# if [ -d "$HOME/.sel4_cache" ]
+# then
+#     CACHE_DIR="$HOME/.sel4_cache"
+# else
+#     CACHE_DIR="$SCRIPT_PATH/.sel4_cache"
+# fi
 
 if [ -e "$SCRIPT_PATH/CMakeLists.txt" ]
 then
     # If we have a CMakeLists.txt in the top level project directory,
     # initialize CMake.
-    cmake -DCMAKE_TOOLCHAIN_FILE="$SCRIPT_PATH"/kernel/gcc.cmake -G Ninja "$@" \
-        -DSEL4_CACHE_DIR="$CACHE_DIR" -C "$SCRIPT_PATH/settings.cmake" "$SCRIPT_PATH"
+    CCACHE_DISABLE=1 cmake -DCMAKE_TOOLCHAIN_FILE="$SCRIPT_PATH"/kernel/gcc.cmake -G Ninja "$@" \
+        #-DSEL4_CACHE_DIR="$CACHE_DIR" -C "$SCRIPT_PATH/settings.cmake" "$SCRIPT_PATH"
+        -C "$SCRIPT_PATH/settings.cmake" "$SCRIPT_PATH"
 else
     # If we don't have a CMakeLists.txt in the top level project directory then
     # assume we use the project's directory tied to easy-settings.cmake and resolve
@@ -51,5 +53,6 @@ else
     real_easy_settings="$(realpath $SCRIPT_PATH/easy-settings.cmake)"
     project_dir="$(dirname $real_easy_settings)"
     # Initialize CMake.
-    cmake -G Ninja "$@" -DSEL4_CACHE_DIR="$CACHE_DIR" -C "$project_dir/settings.cmake" "$project_dir"
+    #cmake -G Ninja "$@" -DSEL4_CACHE_DIR="$CACHE_DIR" -C "$project_dir/settings.cmake" "$project_dir"
+    CCACHE_DISABLE=1 cmake -G Ninja "$@" -C "$project_dir/settings.cmake" "$project_dir"
 fi
